_schemaV2.graphql

```diff
@@ -6079,7 +6079,7 @@ input CreateUserAddressInput {
 type CreateUserAddressPayload {
   # A unique identifier for the client performing the mutation.
   clientMutationId: String
-  me: Me!
+  me: Me
```

Comment:
haha I was just commenting in the other place that I liked the non-nullability.

I guess when making a mutation request, you usually request the error object _and_ `me` simultaneously, and obviously you don't know ahead of time what it'll be, so if there is an error it'll complain about `me` not being set? /shrug